web/src/components/speaker/Links.vue

```diff
@@ -40,6 +44,10 @@ export default {
       type: String,
       default: '',
     },
+    priorPresentation: {
```

Comment:
We'll need to pass this prompt in on the <links> component from the parent as well.

Reply:
@ann-kilzer thanks for catching this and my apologies for the back and forth.